.github/workflows/check-status.yml

```
@@ -15,6 +15,8 @@ on:
       - release-*
 
 jobs:
+  context:
```

actionlint (1.7.4)

"steps" section is missing in job "context"

(syntax-check)